cookbook/assetic/uglifyjs.rst

```diff
@@ -73,8 +73,8 @@ your JavaScripts:
         assetic:
             filters:
                 uglifyjs2:
-                    # the path to the uglifyjs executable
-                    bin: /usr/local/bin/uglifyjs
+                    # the path to the uglifyjs Node module
+                    bin: ./app/Resources/node_modules/uglify-js/bin/uglifyjs
```

Comment:
1. `node_modules` is like `vendor`, are you ready to put your vendor in this place ?
2. By default `node_module` is not in this place